package.json

```diff
@@ -4,9 +4,9 @@
   "description": "JSON Web Key for JavaScript",
   "main": "src/index.js",
   "scripts": {
-    "test": "./node_modules/.bin/_mocha test -w",
+    "test": "./node_modules/.bin/nyc ./node_modules/.bin/_mocha test",
```

Comment:
I'm pretty sure we can just do `"test": "nyc mocha test"` here.

Reply:
Doesn't that require the packages to be globally installed?

Reply:
I don't think so? (could be wrong of course). I don't have either globally installed, and it works for me.

Reply:
Tested and this is indeed the case. Updated `package.json` accordingly.